meta-printnanny/conf/layer.conf

```diff
@@ -9,5 +9,5 @@ BBFILE_COLLECTIONS += "printnanny"
 BBFILE_PATTERN_printnanny = "^${LAYERDIR}/"
 BBFILE_PRIORITY_printnanny = "10"
 
-LAYERDEPENDS_printnanny = "bitsy core meta-initramfs yocto meta-microcontroller multimedia-layer networking-layer meta-python swupdate raspberrypi meta-tensorflow webserver"
+LAYERDEPENDS_printnanny = "bitsy core meta-initramfs yocto meta-microcontroller multimedia-layer networking-layer meta-python swupdate raspberrypi webserver"
```

Comment:
building the fat static lib from meta/neural-network instead
https://github.com/nnstreamer/meta-neural-network/blob/master/recipes-tensorflow/tensorflow-lite/tensorflow-lite_2.9.1.bb